Makefile.source.mjs

```diff
@@ -389,6 +389,73 @@ target["test-cov"] = function () {
   );
 };
 
+function getParserTestsCommit(id) {
+  const content = readFileSync("./Makefile", "utf8");
+  const commit = content.match(new RegExp(`${id}_COMMIT = (\\w{40})`))[1];
```

Comment:
Do we still need to store these commits in `Makefile`?

Reply:
I tend to keep them so that we don't need to regenerate `Makefile.js`.